### DISCUSSION TITLE
Small new company - need help setting up asset tracking.

### DISCUSSION THREAD

**Discussion by brianpatterson790** [snipe/snipe-it#9392] [Q&A]
Hello!

We're a small company in need of a solution for asset tracking.  We are in the water restoration business and carry a lot of fans, dehumidifiers, water-extracting machines, etc...  Currently, we have a warehouse and 4 vans each containing assets in them that are serialized and can be barcoded.

We need help in setting up a system where we can audit the warehouse and the 4 vans (additional vans in the future) at any time. We also need to be able to tranfers items in and out or a checkout system of some sort. To explain this better, imagine van1 goes to a job site that needs 7 equipment pieces there for 2 days.  We need to be able to checkout the 7 pieces and track that they are currently sitting at the job site and not inside van1.

Willing to pay someone to help us set this up.  Please get in touch with us!

Thanks,
Brian

**Comment by Zomagoras** [comment]
You can pay the Snipe-IT creators to host the software for you: https://snipeitapp.com/pricing

**Comment by kabangaba** [comment]
Email me at michael@2logicstudios.com

> Hello!
> 
> We're a small company in need of a solution for asset tracking. We are in the water restoration business and carry a lot of fans, dehumidifiers, water-extracting machines, etc... Currently, we have a warehouse and 4 vans each containing assets in them that are serialized and can be barcoded.
> 
> We need help in setting up a system where we can audit the warehouse and the 4 vans (additional vans in the future) at any time. We also need to be able to tranfers items in and out or a checkout system of some sort. To explain this better, imagine van1 goes to a job site that needs 7 equipment pieces there for 2 days. We need to be able to checkout the 7 pieces and track that they are currently sitting at the job site and not inside van1.
> 
> Willing to pay someone to help us set this up. Please get in touch with us!
> 
> Thanks,
> Bria
